integrations/model-training/sagemaker/log_completed_sagemaker_runs/README.md

```diff
@@ -0,0 +1,91 @@
+# Logging Sagemaker runs to Comet
+
+Comet supports transferring data from Sagemaker runs to Comet's Experiment Management tool. This approach requires no changes
+to existing Sagemake code. It simply transfers data from completed Sagemaker runs to Comet.
+
+There are three ways to log data from a completed Sagemaker Run to Comet
+
+### 1. Using the Estimator object
+
+Teh first method involves passing the Sagemaker `estimator` object directly into Comet's `log_sagemaker_training_job_v1` utility function.
+
+```python
+from comet_ml.integration.sagemaker import log_sagemaker_training_job_v1
+
+COMET_API_KEY = "<Your Comet API Key>"
+COMET_WORKSPACE = "<Your Comet Workspace>"
+COMET_PROJECT_NAME =  "Your Comet Project Name"
+
+log_sagemaker_training_job_v1(
+    estimator,
+    api_key=COMET_API_KEY,
+    workspace=COMET_WORKSPACE,
+    project_name=COMET_PROJECT_NAME
+)
+```
+
+### 2. Using the Training Job Name
+
+```python
+from comet_ml.integration.sagemaker import log_sagemaker_training_job_v1
+
+COMET_API_KEY = "<Your Comet API Key>"
+COMET_WORKSPACE = "<Your Comet Workspace>"
+COMET_PROJECT_NAME =  "Your Comet Project Name"
+
+TRAINING_JOB_NAME = "<Your training job name>"
+
+log_sagemaker_training_job_by_name_v1(
+    TRAINING_JOB_NAME,
+    api_key=COMET_API_KEY,
+    workspace=COMET_WORKSPACE,
+    project_name=COMET_PROJECT_NAME
+)
+```
+
+### 3. Automatically log data from the last completed Training Job
+
+```python
+from comet_ml.integration.sagemaker import log_last_sagemaker_training_job_v1
+
+COMET_API_KEY = "<Your Comet API Key>"
+COMET_WORKSPACE = "<Your Comet Workspace>"
+COMET_PROJECT_NAME =  "Your Comet Project Name"
+
+log_last_sagemaker_training_job_v1(
+    api_key=COMET_API_KEY,
+    workspace=COMET_WORKSPACE,
+    project_name=COMET_PROJECT_NAME
+)
+```
+
+**Known Limitations:**
+
+- Data transfer is only compatible with Training Jobs that use Sagemaker's [builtin algorithms](https://docs.aws.amazon.com/sagemaker/latest/dg/algos.html)
+- This method only supports logging the following information from Sagemaker
+  - Hyperparameters
+  - Metrics
+  - Sagemaker specific metadata (BillableTimeInSeconds, TrainingImage, etc)
+  - Sagemaker notebook code
+- Real time data logging is not supported from the Sagemaker job
+- Metrics are logged based on wall clock time. Step/Epoch information is not captured
+
+
+For more information, please refer to our [Sagemaker Documentation](https://www.comet.com/docs/v2/api-and-sdk/python-sdk/reference/integration.sagemaker/)
+
+
+## Run the Example
+
+To run this example, you will need both a Sagemaker account and a [Comet account](https://comet.com/signup)
+
+1. Upload the `mnist.py` and `train_mnist.ipynb` to your Sagemaker Notebook instance.
```

Comment:
It looks easy enough to clone the comet-examples Git repository from Sagemaker studio or hosted JupyterLab, maybe we should recommend cloning the repository instead?

Reply:
Hmm it's a little tricky. Because the user would have to open up a Jupyterlab UI in the Sagemaker. Then open the terminal, then clone `comet-examples` and then navigate to the appropriate directory before being able to open the notebook. 

Might be easier to just upload the folders to a notebook instance.